packages/blockchain/algo/src/lib/services/algo.tx.ts

```diff
@@ -351,72 +502,85 @@ export const algoTx = (args: { algoWeb: AlgoWeb }) => {
       send: {
         /**
          * Send Algorand create FT transaction to the blockchain. This method broadcasts signed transaction to the blockchain.
-         * @param testnet mainnet or testnet version
          * @param body content of the transaction to broadcast
+         * @param testnet mainnet or testnet version
          * @param provider url of the Algorand Server to connect to. If not set, default public server will be used.
          * @returns transaction id of the transaction in the blockchain.
          */
         createFTSignedTransaction: async (
           body: DeployErc20 | DeployErc20KMS,
           testnet = false,
           provider?: string,
-        ) =>
-          AlgorandService.algoandBroadcast({
-            txData: await prepareCreateFTSignedTransaction(body, testnet, args.algoWeb, provider),
-            ...(isWithSignatureId(body) && { signatureId: body.signatureId }),
-          }),
+        ) => {
+          const txData = await prepareCreateFTSignedTransaction(body, testnet, args.algoWeb, provider)
```

Comment:
same as bellow - if signatureId is present - send to API method with body - dont perform any local operations